lib/flatpak-installation.c

```diff
@@ -1998,7 +2034,8 @@ flatpak_installation_uninstall (FlatpakInstallation    *self,
                               cancellable, error))
     return FALSE;
 
-  flatpak_dir_prune (dir_clone, cancellable, NULL);
+  if ((flags & FLATPAK_UNINSTALL_FLAGS_NO_PRUNE) == 0)
```

Comment:
Nitpick: Same comment about using `!` rather than `== 0` here.

Reply:
Same reply: let's see what ends up being merged upstream, and go with that